internal/lsp/types/types.go

```diff
@@ -0,0 +1,6 @@
+package types
+
+type ExecuteCommandParams struct {
```

Comment:
For now, I have only moved this type here. In a follow on PR, I'll move them all here.